blink/blink-shell.html

```diff
@@ -28,6 +29,8 @@
     </style>
     <!-- BLINK SPECIFIC: Enable SharedArrayBuffer -->
     <script src="coi-serviceworker.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/xterm@5.1.0/lib/xterm.min.js"></script>
```

Comment:
Could you please download this JS blob into third_party/ and put the contents of https://github.com/xtermjs/xterm.js/blob/master/LICENSE in a comment at the top? Please also put the URL from which it was obtained into the comment. The same goes for the .css file.

Reply:
Done. I'm not sure whether that's technically enough though, because that file itself is built from (and thus contains) multiple other JS sources... With the way the node ecosystem works we'd probably end up with ~1000 license files.

Reply:
I think this change is a step in the right direction to ensuring we respect the license terms of open source authors. We would ideally vendor the transitive closure of dependencies and setup a build system to compile all this typescript code, however I didn't want to bog you down in process. Contributions are welcome in helping with npm dependency analysis to ensure everyone who contributed is represented. What's most important is that we're minimizing the number of third party web servers our app needs to contact, thereby offering better security and privacy to the users of our code.